models/public/erfnet/README.md

```diff
@@ -0,0 +1,114 @@
+# ERFNet
+
+## Use Case and High-Level Description
+
+  This is a ONNX* version of `erfnet` model designed to perform real-time lane detection.
+  This model is pre-trained in PyTorch* framework and retrained by CULane.
+  For details see [repository](https://github.com/Zhangxianwen2021/ERFNet),
+  paper of [ERFNet](https://doi.org/10.1109/TITS.2017.2750080) and [repository](https://github.com/cardwing/Codes-for-Lane-Detection/tree/master/ERFNet-CULane-PyTorch)
+
+## Specification
+
+| Metric            |        Value          |
+|-------------------|-----------------------|
+| Type              | Semantic segmentation |
+| GFLOPs            | 11.13                 |
+| MParams           | 7.87                  |
+| Source framework  | PyTorch*              |
+
+## Accuracy
+
+|  Metric  |  Value |
+|  ------  | -------|
+| mean_iou | 76.47% |
+
+## Input
+
+### Original model
+
+Image, name - `input_1`, shape - `1,3,208,976`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Channel order is `RGB`.
+Scale value - 255.
+
+### Converted model
+
+Image, name - `input_1`, shape - `1,3,208,976`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Channel order is `BGR`.
+
+## Output
+
+### Original model
+Image, name - `output1`, shape - `1,5,208,976`, format is `B,C,H,W` where:
```

Comment:
```suggestion
Feature map, name - `output1`, shape - `1,5,208,976`, format is `B,C,H,W` where:
```

Reply:
also it will be nice if you provide information about how to interpret output, something like here:
https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel/road-segmentation-adas-0001#outputs